tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -113,6 +114,27 @@ def post_optimize_action(self):
         self._g.update_proto()
         self._g.topological_sort(self._g.get_nodes())
 
+    def merge_transpose_with_same_input(self):

```

Comment:
a better name? technically, not just same input, also same perm.
e.g. merge_duplicated_transpose?